php/helpers.php

```diff
@@ -172,13 +188,18 @@ function reset_block_rows( $name ) {
  * @return int|bool The total amount of rows. False if the repeater isn't found.
  */
 function block_row_count( $name ) {
-	global $block_lab_attributes;
+	/**
+	 * An array containing the block attributes.
+	 *
+	 * @var array $attributes
+	 */
+	$attributes = block_lab()->loader->get_data( 'attributes' );
 
-	if ( ! isset( $block_lab_attributes[ $name ]['rows'] ) ) {
+	if ( ! $attributes || ! isset( $attributes[ $name ]['rows'] ) ) {
```

Comment:
`! isset( $attributes[ $name ]['rows'] )` should be enough, as it'll also be false if `! $attributes`

Reply:
Done in https://github.com/getblocklab/block-lab/pull/435/commits/0fbf45adb15a4ce1a8fd10112939b68a67249365